python/lsst/meas/modelfit/display/cModelDisplay.py

```diff
@@ -0,0 +1,264 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+import lsst.afw.image as afwImage
+import lsst.meas.modelfit as measMod
+import lsst.shapelet as shapelet
+
+
+def displayReconstruedCmodel(exposure, record, cModelConfig, display="mpl"):
+    """ Display an image, the Cmodel, and residuals
+
+    Parameters
+    ----------
+    exposure : `lsst.afw.image.Exposure`
+        Exposure object that contains the source which was  modeled
+    record : `lsst.afw.table.SourceRecord`
+        Record object which contains the measurements made on the source
+    cModelConfig : `lsst.meas.modelfit.CModel(SingleFrame/Forced)Config`
+        Configuration object of the CModel plugin used in the measurement
+        process
+    display : `str`, optional
+        Display in which to render the data, may be mpl for matplotlib or afw
+        for afwDisplay, defaults to mpl
+
+    Raises
+    ------
+    `NotImplementedError`
+        If the display backend specified is not implemented
```

Comment:
But you shouldn't be setting a backend.  The user can set the default backend.

This is true for afwDisplay (which you should probably be using, but that's a different ticket), but it's also true for matplotlib.